src/machines/java/com/enderio/machines/client/gui/widget/CapacitorEnergyWidget.java

```diff
@@ -42,8 +42,9 @@ public void renderCapacitorTooltip(GuiGraphics guiGraphics, int mouseX, int mous
         List<Component> list = new ArrayList<>();
         list.add(EIOLang.NOCAP_TITLE.withStyle(ChatFormatting.DARK_AQUA));
         String[] split = EIOLang.NOCAP_DESC.getString().split("\n");
-        list.add(Component.literal(split[0].stripTrailing()));
-        list.add(Component.literal(split[1].stripLeading()));
+        for (String s :split) {
```

Comment:
Looks like this should fix #472, if so can we link the PR to the issue so it gets closed :)